### ISSUE TITLE
Migrating from KookieStocks 

### ISSUE THREAD

**Issue by GrimorgGH** [darthpilou/EZStock#2]
I was using KookieStocks until a few days ago, when I learned about your fork. Something isn't working as expected, I always see the stock panel as follows:

![ezstock](https://user-images.githubusercontent.com/74399345/99040097-cf61b880-2588-11eb-97db-9ed8d389a8c4.PNG)

0.1 opacity, no highlights based on price fluctuation.
I'm also using Chorizorro fork of CookieMonster (https://chorizorro.github.io/CookieMonster/), no other addons.

**Comment by darthpilou** [comment]
Hey, the thing is the tool record the lowest and highest value ever recorded and there's a "security" to not highlight anything until there's enough difference between lowest and highest (I think it's 30). So you either have to wait until it has seen enough, either you can modify your history manually.

I'll paste you the code to update with my values if you want to. Just copy paste in the console and execute.
I haven't really publicized this project because I know there are issues when ascending, it freaks out because all stock reset even though it never saw you sell them, etc.

And for your info, you can automate buying selling, with the command EZStock.automate = true; I use it, but it probably needs some more testing too, so use at your own risk.

Commands to update low/high :

EZStock.goods[0].lowval = 1.1;
EZStock.goods[1].lowval = 2;
EZStock.goods[2].lowval = 1.8;
EZStock.goods[3].lowval = 2;
EZStock.goods[4].lowval = 2.2;
EZStock.goods[5].lowval = 2.2;
EZStock.goods[6].lowval = 2.8;
EZStock.goods[7].lowval = 2.8;
EZStock.goods[8].lowval = 2.7;
EZStock.goods[9].lowval = 3.2;
EZStock.goods[10].lowval = 3.3;
EZStock.goods[11].lowval = 4.7;
EZStock.goods[12].lowval = 7;
EZStock.goods[13].lowval = 17.7;
EZStock.goods[14].lowval = 26;
EZStock.goods[15].lowval = 41.1;
EZStock.goods[0].highval = 59.8;
EZStock.goods[1].highval = 90.3;
EZStock.goods[2].highval = 110.6;
EZStock.goods[3].highval = 131.8;
EZStock.goods[4].highval = 103.1;
EZStock.goods[5].highval = 134.4;
EZStock.goods[6].highval = 138.2;
EZStock.goods[7].highval = 132.6;
EZStock.goods[8].highval = 142.4;
EZStock.goods[9].highval = 145.6;
EZStock.goods[10].highval = 155;
EZStock.goods[11].highval = 165.5;
EZStock.goods[12].highval = 179.4;
EZStock.goods[13].highval = 188.3;
EZStock.goods[14].highval = 197.1;
EZStock.goods[15].highval = 196.7;

Greets

**Comment by GrimorgGH** [comment]
Yes, I've understand a bit more since publishing this issue, and your feedback confirms what I thought.
I have a couple of suggestions: 

1. With opacity 0.1 the panel is almost unreadable, I'd raise this value.
2. I'm not sold about the way the calculations are done; instead of using the history of high/low values why don't you refer to the resting values (see [this](https://cookieclicker.fandom.com/wiki/Stock_Market))?

Thank you.
